src/useCases/__tests__/PreLoadUseCase.test.ts

```diff
@@ -0,0 +1,36 @@
+import * as E from 'fp-ts/Either';
+import * as RA from 'fp-ts/ReadonlyArray';
+import { makeLogger } from '../../logger';
+import { PreLoadUseCase } from '../PreLoadUseCase';
+import { config } from '../../__tests__/data';
+import * as inMemory from '../../adapters/inMemory';
+import { pipe } from 'fp-ts/lib/function';
+
+const logger = makeLogger();
+const repository = inMemory.makeRepository(logger)([]);
+const body = [
+  {
+    preloadIdx: '0',
+    contentType: 'application/pdf',
+    sha256: 'xxx',
+  },
+];
+
+describe('PreLoadUseCase', () => {
+  it('should return the key into url for each elements', async () => {
+    const [apiKey] = ['key-value'];
+    const actual = await PreLoadUseCase(logger, config.server.uploadToS3URL, repository)(apiKey)(body)();
+    const checkKey = pipe(
+      actual,
+      E.exists(
+        (e) =>
+          e.statusCode === 200 &&
+          pipe(
+            e.returned,
+            RA.every((e) => e.url.endsWith(e.key))
+          )
+      )
+    );
```

Comment:
```suggestion
      E.exists(
        (response) =>
          response.statusCode === 200 &&
          pipe(
            response.returned,
            RA.every((e) => e.url.endsWith(e.key))
          )
      )
    );
```
just a suggestion to improve the readability of the elements (in the inner pipe there is another variable `e`. it's in a different scope, but still very close to the `e` element in the outer scope.